Makefile

```diff
@@ -43,12 +44,38 @@ clean:
 $(BIN):
 	-mkdir -p $(BIN)
 
-# Primary binary - special deps
-$(BIN)/deepseg: $(TFLIBS)/libtensorflow-lite.a deepseg.cc loopback.cc transpose_conv_bias.cc
+# Primary binaries - special deps
+$(BIN)/deepseg: app/deepseg.cc $(BIN)/libbackscrub.a $(BIN)/libvideoio.a
 	g++ $^ ${CFLAGS} ${TFCFLAGS} ${LDFLAGS} ${TFLDFLAGS} -o $@
 
+# Unusual archive munging here is the nicest way to ensure we have Tensorflow Lite & our library code
+# easily accessible through one static library
+$(BIN)/libbackscrub.a: $(BIN)/libbackscrub.o $(BIN)/transpose_conv_bias.o $(TFLIBS)/libtensorflow-lite.a
+	cp -p $(TFLIBS)/libtensorflow-lite.a $@
+	ar rv $@ $(BIN)/libbackscrub.o $(BIN)/transpose_conv_bias.o
```

Comment:
Why not leave them separate and just pass both of them to the linker later on? If we pass `libbackscrub.a` followed by its dependencies, we can avoid this hack. As an aside: If we want to skip managing libs ourselves we could ask `libtool` to do that job for us.

Reply:
I did this originally to avoid having to pass transitive dependencies back to projects that use `libbackscrub.a`, not to fix anything in _this project_. However, since I think the `make` build herein is never going to be used for other projects, and the `cmake` build can export dependencies then I'll remove this hack as suggested.